composer.json

```diff
@@ -16,7 +16,7 @@
     }
   ],
   "require": {
-    "yiisoft/yii2": "*"
+    "yiisoft/yii2": "^2.0.13"
```

Comment:
this limits it to version 2.0. 

But probably it is to also work in upcoming verison, should we change it to
`"yiisoft/yii2": ">=2.0"`

wdyt? (I am sure about this one, because not really following current development)

Reply:
Yes. How to change it? Should I create another pull-request?

Reply:
updated it